client/src/layouts/AppLayout.jsx

```diff
@@ -1,30 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Container } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-import Header from '../components/Header'
-import { Container } from '@material-ui/core'
 
-const useStyles = makeStyles({
-  root: {
-    paddingTop: '100px',
-  },
+import Header from '../components/Header';
+
+const useStyles = makeStyles((theme) => {
+  const sidebarBorder = '2px solid #E9EDFA';
```

Comment:
Should we use a variable of some sort instead of hard-coded value for the color?

Reply:
Fixed!
That one slipped through the fingers of copy-paste :P